L/LibCURL/LibCURL@8/build_tarballs.jl

```diff
@@ -1,5 +1,3 @@
 include("../common.jl")
 
-build_libcurl(ARGS, "LibCURL", v"8.14.1")
-
-# Build trigger: 1
+build_libcurl(ARGS, "LibCURL", v"8.15.0")
```

Comment:
You need to edit this, otherwise the package won't be rebuilt: https://github.com/JuliaPackaging/Yggdrasil/blob/d878fa3b48744b379a969bfbaea12fdc57fdc206/CONTRIBUTING.md#understanding-build-cache-on-yggdrasil.  This would have failed because of https://github.com/JuliaPackaging/Yggdrasil/pull/11702#discussion_r2217953134

Reply:
Thanks.